src/components/MovieList/MovieList.tsx

```diff
@@ -107,29 +149,46 @@ const MovieList: FunctionComponent = () => {
     }
   }, [moreRecentList]);
 
+  const renderImage = (path?: string, altText?: string) => {
+    const localPath = path 
+      ? `https://image.tmdb.org/t/p/w500${path}` 
```

Comment:
Maybe we should turn this base URL for images into a global variable since it is used in multiple parts of the website. Thoughts?

Reply:
I agree with that! 👍